types/index.tsx

```diff
@@ -0,0 +1,14 @@
+export interface AuthorProps {
+  name: string
+  picture: any
```

Comment:
Not quite sure what the correct type here is

Reply:
I would think it's time `image` perhaps? Or was that changed? 🤔

Reply:
This is technically a:
```
import {Image} from '@sanity/types'
```
But let's deal with this in a follow-up as it's not clear how to strike the best balance between:
- A: good detailed types that make it a lot of work to change your groq queries.
- B: friendly types that let you easily change groq queries, but TS can't know if the data groq returns matches what the components expect.